lib/fs.js

```diff
@@ -214,7 +214,12 @@ fs.exists = function(path, callback) {
   req.oncomplete = cb;
   binding.stat(pathModule._makeLong(path), req);
   function cb(err, stats) {
-    if (callback) callback(err ? false : true);
+    if (callback) {
+      if (callback.length > 1)
+        callback(null, err ? false : true);
+      else
+        callback(err ? false : true);
```

Comment:
Can we put a deprecation warning when using the single argument callback? Should this be implemented right away or in a future version?